config.go

```diff
@@ -8,86 +8,66 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-// ConfigFileName is an optional file checked for in the function root.
-const ConfigFileName = ".faas.yaml"
+// ConfigFileName is the name of the config's serialized form.
+const ConfigFileName = ".faas.config"
```

Comment:
Should this be kept .yaml?

Reply:
I think it could stay yaml, but if you are going to change that, don't forget change it also `contentiousFiles`.

Reply:
I think keeping it `.yaml` makes sense.

Reply:
Makes sense to me to, changed it back.